ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java

```diff
@@ -48,14 +43,11 @@ public void XSSOnPhoneNumberFieldTest() throws Exception {
         registrationEditSectionPage.clearPhoneNumber();
         registrationEditSectionPage.enterPhoneNumber("<script>alert(0)</script>");
         registrationEditSectionPage.clickOnConfirmEdit();
-        assertThat(registrationEditSectionPage.getValidationErrors(), is(not(empty())));
+        assertTrue(registrationEditSectionPage.getInvalidPhoneNumberNotification().contains(("Must be a valid phone number (with +, -, numbers or parentheses)")));
         registrationEditSectionPage.clearPhoneNumber();
         registrationEditSectionPage.enterPhoneNumber("111111111");
         registrationEditSectionPage.clickOnConfirmEdit();
         patientDashboardPage = registrationEditSectionPage.confirmPatient();
-        //Ignored as show contact may hide under success message
-        //patientDashboardPage.clickOnShowContact();
-        //assertThat(patientDashboardPage.getTelephoneNumber(), is("111111111"));
```

Comment:
I think the climax of test logic should `assert that` the entered telephone number is available in the `db instance` as implemented at code line number `58.` Otherwise removing code line `57` and `58` makes the test tasteless(meaningless). There is a need to investigate why Show contact may some times hide under success message, once its fixed then code line `57 `and `58 `should be  uncommitted to perform their logic!

Reply:
> Otherwise removing code line `57` and `58` makes the test tasteless(meaningless)

Uhmm Meaningless! , you mean this test was to capture availability of phone number or capturing the xss bug in phone number input tag?, this test is not capturing phone number, the main point of this test is to handle/ track xss bug in the phone number input.